codegens/curl/lib/index.js

```diff
@@ -236,6 +239,13 @@ self = module.exports = {
         default: true,
         description: 'Automatically follow HTTP redirects'
       },
+      {
+        name: 'Follow original HTTP method',
+        id: 'followOriginalHttpMethod',
+        type: 'boolean',
+        default: false,
+        description: 'Use the original HTTP method when following redirects'
```

Comment:
![image](https://user-images.githubusercontent.com/16756296/216244990-72ed1c4d-2b92-4684-a352-4826efe35443.png)
I think we can keep the name and description the same as the request setting option.